app/controllers/orders/conversation.js

```diff
@@ -1,26 +1,26 @@
-import config from '../../config/environment';
-import detail from './detail';
-import Ember from 'ember';
+import config from "../../config/environment";
+import detail from "./detail";
+import Ember from "ember";
 
 export default detail.extend({
   messagesUtil: Ember.inject.service("messages"),
   isPrivate: false,
-  order: Ember.computed('model', function(){
-    return this.get('model');
+  order: Ember.computed("model", function() {
```

Comment:
you could have directly used alias ?

Reply:
sure 👍

Reply:
using alias, and re-committing in 5 mins.